themes/hugo-tutorials/layouts/partials/runnable.html

```diff
@@ -82,8 +82,8 @@ <h5 class="modal-title" id="runnable-url-modal-label">Change Dgraph Alpha URL</h
               <label for="inputDgraphUrl">Dgraph Alpha Address</label>
               <input type="text" class="form-control" id="inputDgraphUrl" placeholder="http://localhost:8080" style="padding:0 4px;">
               <!-- <small id="serverStatus" class="form-text text-muted">Server status goes hear.</small> -->
-              <label for="inputSlashKey">Slash API Key</label>
-              <input type="password" class="form-control" id="inputSlashKey" placeholder="" style="padding:0 4px;">
+              <label for="inputDgraph CloudKey">Dgraph Cloud API Key</label>
+              <input type="password" class="form-control" id="inputDgraph CloudKey" placeholder="" style="padding:0 4px;">
             </div>
```

Comment:
This is a breaking change. I will review this again on Monday

Reply:
The id space makes a difference in the javascript

Reply:
Thanks for catching this; I changed this back to avoid a breaking change. No need to change any instances of "Slash" that aren't visible to a reader.